packages/commcare/src/Adaptor.js

```diff
@@ -347,6 +348,119 @@ export function request(method, path, body, options = {}) {
     }
   };
 }
+
+/**
+ * Bulk upload data to CommCare. Accepts an array of objects, converts them into
+ * an XLS representation, and uploads.
+ * @public
+ * @function
+ * @example <caption>Upload a single row of data for case-data</caption>
+ * bulk(
+ *    [
+ *      {name: 'Mamadou', phone: '000000'},
+ *    ],
+ *    {
+ *      case_type: 'student',
+ *      search_field: 'external_id',
+ *      create_new_cases: 'on',
+ *    }
+ * )
+ * @example <caption>Upload a single row of data for a look-up table</caption>
```

Comment:
Can this example be made shorter? Do we really need quite so many fields in the data?

Reply:
Something like this perhaps 
```js
bulk("lookup-table", {
  types: [
    {
      "field 1": "Company",
      "field 2": "Func",
    },
  ],
  obat: [
    {
      "field:Company": "OpenFn",
      "field:Func": "Workflow Automation",
    },
  ],
});
```

Reply:
I can use a different example but yes it can get as long as the keys in the lookup-table

Reply:
Yeah sure, as long as it's lookup-table